corehq/warehouse/management/commands/create_batch.py

```diff
@@ -1,52 +1,26 @@
 from __future__ import absolute_import
-from django.core.management import BaseCommand, CommandError
-from dimagi.utils.parsing import string_to_utc_datetime
+from datetime import date
+from django.core.management import BaseCommand
 from corehq.warehouse.models import Batch
 
 
-USAGE = """Usage: ./manage.py create_batch <batch_id> -s <start_datetime> -e <end_datetime>
+USAGE = """Usage: ./manage.py create_batch
 """
 
 
 class Command(BaseCommand):
     """
-    Example: ./manage.py create_batch 222617b9-8cf0-40a2-8462-7f872e1f1344 -s 2017-05-01 -e 2017-06-01
+    Example: ./manage.py create_batch
     """
     help = USAGE
 
-    def add_arguments(self, parser):
-        parser.add_argument('batch_id')
+    def handle(self, **options):
+        last_batch = Batch.objects.filter(completed_on__isnull=False).order_by('-end_datetime').first()
```

Comment:
This makes sense in steady state, but how does the first batch get into the DB? Does this need to check for that scenario?

Reply:
yeah if it doesn't find one we'll need to set the start_datetime to a date that's far enough in the past to capture all source data